src/api/flaskr/service/learn/routes.py

```
@@ -0,0 +1,295 @@
+                    shifu_bid=shifu_bid,
+                    outline_bid=outline_bid,
+                    user_bid=user_bid,
+                    input=input,
+                    input_type=input_type,
+                    reload_generated_block_bid=reload_generated_block_bid,
+                    preview_mode=preview_mode,
+                ),
+                headers={"Cache-Control": "no-cache"},
+                mimetype="text/event-stream",
+            )
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)